src/components/windows-motion-controls.js

```diff
@@ -0,0 +1,382 @@
+var bind = require('../utils/bind');
+var registerComponent = require('../core/component').registerComponent;
+var controllerUtils = require('../utils/tracked-controls');
+var utils = require('../utils/');
+
+var debug = utils.debug('components:windows-motion-controls:debug');
+var warn = utils.debug('components:windows-motion-controls:warn');
+
+// TODO: Point to final glb assets once they are hosted. For now, place them in this hard coded directory.
+var MODEL_BASE_URL = '/examples/assets/models/controllers/wmr/';
```

Comment:
As you pointed out we should host the models in the CDN https://github.com/aframevr/assets/tree/gh-pages/controllers

Reply:
For clarity, to get the models in the CDN, I believe one does a pull request against aframevr/assets with the appropriate changes.

Reply:
Correct! Just a PR against the `gh-pages` branch on https://github.com/aframevr/assets/tree/gh-pages/controllers

Reply:
`gh-pages` so they get deployed on the CDN

Reply:
not unless something changed?  for example this is vive tracker https://github.com/aframevr/assets/pull/21

Reply:
specific example for reference, this was vive-tracker: https://github.com/aframevr/assets/pull/21